src/components/DayPicker.jsx

```diff
@@ -190,27 +242,31 @@ export default class DayPicker extends React.Component {
   }
 
   adjustDayPickerHeight() {
-    const $transitionContainer = $(ReactDOM.findDOMNode(this.refs.transitionContainer));
-    const heights = $transitionContainer.find('.CalendarMonth[data-visible="true"]').get().map(
-      (month) => this.getMonthHeight($(month))
-    );
+    const transitionContainer = ReactDOM.findDOMNode(this.refs.transitionContainer);
+    const heights = [];
+
+    slice.call(transitionContainer.querySelectorAll('.CalendarMonth')).forEach((el) => {
```

Comment:
`Array.prototype.slice` is the way to do it. Thought I was going to be using it more, hence the reference at the top.